lighthouse-core/audits/accessibility/manual/meta-viewport.js

```diff
@@ -28,18 +20,16 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
-class MetaViewport extends AxeAudit {
+class MetaViewport extends ManualAudit {
```

Comment:
sorry @jayaddison when I said run it manually now I meant that it might not get automatically pulled in by our accessibility gatherer so we might have to explicitly ask axe to run it. 

we definitely don't want to lose this as an automatic check in Lighthouse that would be a dealbreaker for upgrading IMO.

Reply:
No problem, I did wonder if I might have been misunderstanding something there and should have clarified.  I'll check the gatherer code and make sure this is enabled at the same time as the version upgrade.